src/main/java/xyz/duncanruns/julti/Julti.java

```diff
@@ -122,6 +122,22 @@ public static boolean isRanFromAlternateLocation() {
         return !Paths.get("").toAbsolutePath().equals(getSourcePath().getParent());
     }
 
+    // Uses portion https://stackoverflow.com/questions/35129457/how-to-check-if-a-process-is-running-on-windows
+    // CMD help from Jojoe, thank you jojoe
+    public static void isWallAHKOpen() throws IOException {
```

Comment:
Does not need to be in Julti class.

Naming implies a boolean return type when it is a void return type.